theme/static/js/hs-vue/subject-keywords-app.js

```diff
@@ -79,6 +79,11 @@ let subjKeywordsApp = new Vue({
                         }
                     }
                 }.bind(this), "json");
+        },
+        safeJS: function (input) {
+            input.replace("\\", "\\\\")
+            console.log(input)
+            return encodeURIComponent(input)
```

Comment:
remove line 85 for log. Also, can this be consolidated with ecsJS() in some way?

Reply:
For now, I'd like to leave it in the new Vue instance that Mauriel has created. In subsequent work, I'll be doing consolidation and improving consistency of approaches.

Reply:
Thanks Hong, made changes in eea7bc5b7